src/main/kotlin/com/openlattice/chronicle/controllers/StudyController.kt

```diff
@@ -543,6 +519,67 @@ class StudyController @Inject constructor(
         return studyService.getStudyParticipantStats(studyId)
     }
 
+    override fun getParticipantsData(
+        studyId: UUID,
+        dataType: ParticipantDataType,
+        participantIds: Set<String>,
+        startDateTime: OffsetDateTime?,
+        endDateTime: OffsetDateTime?
+    ): Iterable<Map<String, Any>> {
+        ensureReadAccess(AclKey(studyId))
+        return when(dataType) {
+            ParticipantDataType.Preprocessed -> TODO("Not implemented")
+            ParticipantDataType.AppUsageSurvey -> downloadService.getParticipantsAppUsageSurveyData(studyId, participantIds, startDateTime, endDateTime)
+            ParticipantDataType.IOSSensor -> {
+                val sensors = getStudySensors(studyId)
+                downloadService.getParticipantsSensorData(studyId, participantIds, sensors, startDateTime, endDateTime)
+            }
+            ParticipantDataType.UsageEvents -> {
+                downloadService.getParticipantsUsageEventsData(studyId, participantIds, startDateTime, endDateTime)
+            }
+        }
+    }
+
+    @Timed
+    @GetMapping(
+        path = [STUDY_ID_PATH + PARTICIPANTS_PATH + DATA_PATH],
+        produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    fun getParticipantsData(
+        @PathVariable(STUDY_ID) studyId: UUID,
+        @RequestParam(value = DATA_TYPE) dataType: ParticipantDataType,
+        @RequestParam(value = PARTICIPANT_ID) participantIds: Set<String>,
+        @RequestParam(value = START_DATE) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) startDateTime: OffsetDateTime?,
+        @RequestParam(value = END_DATE) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) endDateTime: OffsetDateTime?,
+        response: HttpServletResponse
+    ): Iterable<Map<String, Any>> {
+        val data = getParticipantsData(
+            studyId,
+            dataType,
+            participantIds,
+            startDateTime,
+            endDateTime
+        )
+
+        val fileName = "${dataType}_${LocalDate.now().format(DateTimeFormatter.BASIC_ISO_DATE)}.csv"
```

Comment:
It might be a good idea to make the file name an argument from the frontend that's limited to 64 characters and default to LocalData.now()